app/components/works/add_files_component.html.erb

```diff
@@ -74,9 +74,53 @@
     <%= form.radio_button :upload_type, 'globus', class: 'form-check-input', 'data-file-uploads-target': 'globusRadioButton', 'data-deposit-button-target': 'globusRadioButton',
       data: { action: 'complex-radio#disableUnselectedInputs file-uploads#updatePanelVisibility deposit-button#updateDepositButtonStatus'}
     %>
-    <%= form.label :upload_type, "Upload one or more files (>10GB) via Globus web-based transfer software.", class: 'form-check-label fw-semibold' %>
-    <p>Follow <%= link_to "these instructions", Settings.globus.help_doc_url, target: :blank %> to set up your Globus account and the necessary software, then return here to complete your deposit.</p>
+    <%= form.label :upload_type, "Upload one or more files (more than 10GB) which will be displayed as uploaded, including any file hierarchy.", class: 'form-check-label fw-semibold' %>
```

Comment:
So the Globus radio button doesn't mention Globus anymore? Huh. Implementation detail?

Reply:
Yeah, it makes sense to me.  I suspect that testing revealed users don't really care which particular system is used (and probably don't know what Globus even means), so the radio button is now just asking them what they are trying to do (i.e. upload more than 10 GB).  Once you click the radio button, you get all of the details.